ex04.ex

```diff
@@ -75,8 +75,11 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
-
+  def even_odd(xs) do
```

Comment:
### -2

I'm deducting points here because this function does too much. 

I am trying to work the class towards a style where you write using functions, and not just statements in a function. 

~~~ elixir
  def even_odd(list) do
    reduce(list, { [], [] }, &partition_even_odd/2)
    |> reverse_two_lists
  end

  defp partition_even_odd(x, { es, os }) when Integer.is_even(x), do: {[x | es], os}
  defp partition_even_odd(x, { es, os }) when Integer.is_odd(x),  do: {es, [x | os]}

  defp reverse_two_lists({ evens, odds }) do
    { reverse(evens), reverse(odds) }
  end
~~~

I deducted a second mark because your function allows be to write

~~~
even_odd([1, :cat, "你好", 3])  #=> { [], [[1, :cat, "你好", 3]}
~~~